examples/images_contours_and_fields/demo_bboximage.py

```diff
@@ -37,7 +37,7 @@
         not m.startswith(('spectral', 'Vega'))  # Skip deprecated colormaps.
     )
 
-    #fig.subplots_adjust(top=0.99, bottom=0.01, left=0.2, right=0.99)
+    # fig.subplots_adjust(top=0.99, bottom=0.01, left=0.2, right=0.99)
```

Comment:
I guess at some point we should clean this up and just delete those lines of codes.